### DISCUSSION TITLE
Support / Correct Wiring for 3 wire Pacific Electronics 3403 apartment intercom system

### DISCUSSION THREAD

**Discussion by jasonanovak** [PricelessToolkit/ESPBell-LITE#25] [Q&A]
Hi! 

I have a Pacific Electronics 3403 apartment intercom box in my apartment that has three wires going to it:
![image](https://github.com/user-attachments/assets/8c9f1ff7-5230-4a3b-9e19-cf9c4fb496db)

Based on what I've read it seems like, with external power, the ESPBell-LITE _should_ support this if wired as follows:
|Intercom|ESPBell-Lite|
|-|-|
|C|One Pair of Lock Wires|
|T|One Pair of Lock Wires|
|B|Bell 1|
|B|Bell 2|

Is that right?

(I understand from [this doc](http://wiring.alphacommunications.com/AWD103.PDF) that:
- T and C wires are common to all apartments;
- B is per apartment for the individual apartments lobby panel signal wire)

(And I'm basing some of this off of [discussion 7](https://github.com/PricelessToolkit/ESPBell-LITE/discussions/17).)

Thanks!

**Comment by PricelessToolkit** [comment]
Hi, C and T are correct, but I'm not sure about detecting doorbells. I think B should go to Bell1 and C to Bell2, but you need to check if this affects the sound quality, as these are essentially the speaker terminals.
